src/Features/Core/Portable/SymbolSearch/Windows/SymbolSearchUpdateEngine.Update.cs

```diff
@@ -553,9 +554,21 @@ private async Task<XElement> TryDownloadFileAsync(IFileDownloader fileDownloader
 
                 using var reader = XmlReader.Create(stream, settings);
 
-                var result = XElement.Load(reader);
-                await LogInfoAsync("Converting data to XElement completed", cancellationToken).ConfigureAwait(false);
-                return result;
+                try
+                {
+                    var result = XElement.Load(reader);
+                    await LogInfoAsync("Converting data to XElement completed", cancellationToken).ConfigureAwait(false);
+                    return result;
+                }
+                catch (XmlException e)
+                {
+                    // If the XML fails to load, the caller should try again. Since
+                    // this is dependent on file downloader it's possible network
+                    // errors or incomplete file streams. No reason to fail here, just try 
+                    // again
+                    await LogExceptionAsync(e, "Converting data to XElement failed", cancellationToken).ConfigureAwait(false);
+                    return null;
```

Comment:
hrmm... this is bad though.  this indicate the `await fileDownloader.ReadFileAsync().ConfigureAwait(false);` didn't properly work.  It is supposed to abstract away getting this file successfully.  

Note: fileDownloader (owned by platform, not us) will cache its results.  So if we get to this failure point, then we're going to stick in that failure mode perpetually.

Reply:
That is likely why we're reporting millions of exceptions here. I guess the question is: is there follow up from us on this?